src/sentry/static/sentry/app/utils/formatters.tsx

```diff
@@ -57,35 +57,55 @@ export const SECOND = 1000;
 export function getDuration(
   seconds: number,
   fixedDigits: number = 0,
-  abbreviation: boolean = false
+  abbreviation: boolean = false,
+  extraShort: boolean = false
 ): string {
   const value = Math.abs(seconds * 1000);
 
-  if (value >= MONTH) {
+  if (value >= MONTH && !extraShort) {
     const {label, result} = roundWithFixed(value / MONTH, fixedDigits);
     return `${label}${
       abbreviation ? tn('mo', 'mos', result) : ` ${tn('month', 'months', result)}`
     }`;
   }
   if (value >= WEEK) {
     const {label, result} = roundWithFixed(value / WEEK, fixedDigits);
-    return `${label}${abbreviation ? t('wk') : ` ${tn('week', 'weeks', result)}`}`;
+    const units = extraShort
+      ? t('w')
+      : abbreviation
+      ? t('wk')
+      : ` ${tn('week', 'weeks', result)}`;
+    return `${label}${units}`;
```

Comment:
```suggestion
if (extraShort) {
  return `${label}${t('w')}`;
}

if (abbreviation) {
  return `${label}${t('wk')}`;
}

return `${label}${tn('week', 'weeks', result)}`;
```

I'm not a big fan of nested ternaries 😄  it's just a suggestion 😉